From 7cde89d2c8efd4d499de74503f11148a963d2519 Mon Sep 17 00:00:00 2001
From: Uri Mashiach <uri.mashiach@compulab.co.il>
Date: Sun, 24 Nov 2019 11:08:31 +0200
Subject: [PATCH 12/13] sb-imx8x: enable VPU decoding

The video decoding is done by the CPU.
Enable the VPU decoding.

Signed-off-by: Uri Mashiach <uri.mashiach@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/sb-imx8x.dts | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/compulab/sb-imx8x.dts b/arch/arm64/boot/dts/compulab/sb-imx8x.dts
index ad660b058782..756d74c5590d 100644
--- a/arch/arm64/boot/dts/compulab/sb-imx8x.dts
+++ b/arch/arm64/boot/dts/compulab/sb-imx8x.dts
@@ -421,3 +421,12 @@
 		};
 	};
 };
+
+&vpu {
+	status = "okay";
+};
+
+&vpu_decoder {
+	core_type = <1>;
+	status = "okay";
+};
-- 
2.11.0

